src/main/java/co/inventorsoft/StreamHomework.java

```diff
@@ -49,7 +65,11 @@
      * @return map {age : people with this age}
      */
     public Map<Integer, List<Person>> groupByAge(final List<Person> people) {
-        return null;
+
+        Map<Integer, List<Person>> listMap = people.stream()
```

Comment:
It is not necessary to declare new variable just for further returning. You can return the result of stream processing immediately like
```
return something.stream()
  .filter(..)
  .map(..)
  .collect(..);
```

It applies to all of the methods:)

Reply:
Got it!